tests/python_functional/.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+exclude: '^$'
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
I don't know if it is possible, but I would prefer a version that is not based on pre commit hook.

If this was a cli command, then travis could use it for testing.
During local developement, I could also call the cli command to find trouble, and not need to commit to get the same information.

In case I run into trouble many times in regards to these checks, I would probably add it to pre-commit hook. But I would delay adding anything to pre-commit hook as long as possible.

Reply:
Ok, I see. 
I will check how can we run pre-commit for Travis.